### DISCUSSION TITLE
Iris with ReplayMod

### DISCUSSION THREAD

**Discussion by Bean0-0** [IrisShaders/Iris#735] [Ideas]
When I try to render a recording with replaymod it gives me a banner saying rendering is not supported with this version of Sodium
![2021-08-08_18 27 52](https://user-images.githubusercontent.com/72772737/128648816-711eb7b0-37f1-4391-b5b3-651da1d00af1.png)

**Comment by byquanton** [comment]
Replay mod has an own Version of Sodium witch is Compatible with Replay mod and Iris

**Reply by Bean0-0** [reply]
where would I get this?

**Reply by forkiesassds** [reply]
Their fork would have to be merged to upstream so it could work with Iris's fork of sodium.
